account_invoice_validate_send_email/views/res_company_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_company_form" model="ir.ui.view">
+        <field name="name">res.company.form</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='payment_approval_seal_stamp']" position="after">
```

Comment:
This field is pro_mi7_account and we prefer to reduce the dependency on the modules from the pro-xxx repos if anything.  But this module depends on pro_mi7_account anyway, therefore it's OK for now.